### PR TITLE
Charts: don't deploy metallb-excludel2 cm if the speaker is disabled

### PR DESCRIPTION
**Pull request by mustdiechik**
**Is this a BUG FIX or a FEATURE ?**:

/kind cleanup

**What this PR does / why we need it**:

- no need to install "metallb-excludel2" cm, when speaker component is disabled
- added common labels for "metallb-excludel2" cm the same way as for another manifests

**Special notes for your reviewer**:

NONE

**Release note**:

```release-note
The metallb-excludel2 configmap is not deployed from the Helm chart if the speaker is disabled.
```

### CODE REVIEW

**Comment by fedepaol** [comment]
Thanks for the contribution! I see you deleted the PR template, which we are planning to use for crafting the release notes.
Would you mind restoring it and filling the right bits?
Also, would you mind signing the commit?
Also also, can you change the commit subject to what we are doing this commit (something like - don't deploy the exclude l2 config if the speaker is disabled) ?

Thanks again

File: charts/metallb/templates/exclude-l2-config.yaml [hunk 1]
```diff
@@ -1,9 +1,12 @@
+{{- if .Values.speaker.enabled }}
```
Comment (cyclinder) [inline charts/metallb/templates/exclude-l2-config.yaml:1]: we can use the `and` operator here.
```suggestion
{{- if .Values.speaker.enabled and .Values.speaker.excludeInterfaces.enabled }}
```

Comment (mustdiechik) [reply charts/metallb/templates/exclude-l2-config.yaml]: fixed

File: charts/metallb/templates/exclude-l2-config.yaml [hunk 2]
```diff
@@ -20,4 +22,4 @@ data:
     - ^kube-ipvs.*
     - ^cni.*
     - ^nodelocaldns.*
-{{- end }}
\ No newline at end of file
+{{- end }}
```
Comment (cyclinder) [inline charts/metallb/templates/exclude-l2-config.yaml:25]: We should revert this change, we need to end with an extra blank line.

**Comment by cyclinder** [comment]
Thanks for the contribution. And also, Please follow the [comment](https://github.com/metallb/metallb/pull/2298#issuecomment-1963563530) from @fedepaol, Could you mind updating the PR title? so we can easily see what are the PR doing.

**Comment by fedepaol** [comment]
Thanks! Sending to merge

**Comment by fedepaol** [comment]
Also, thanks @cyclinder for the review!
